cms/jinja2/templates/components/streamfield/topic_headline_figures_block.html

```diff
@@ -0,0 +1,17 @@
+<div class="ons-container headline-figures">
```

Comment:
note: DS 72.5.0 has a component for the headline figure: https://github.com/ONSdigital/design-system/pull/3528/files#diff-2a7729f499ad4198c858eb52dede4665c681fad32158edf2396f7143cae17726R4

Reply:
Added here. Jane told me a version without a link does not exist yet, so will replace the other card with the macro once it exists.